### ISSUE TITLE
if not dead - it lacks accessibility features and should be.

### ISSUE THREAD

**Issue by MarcusMorba** [empathyco/vue-carousel#3]
A quick axe DevTools check of the demo start page show´s lot´s of accessibility issues. 
Therefore it would be a good idea to move this code officially into an archive and provide a fully WCAG-compliant and better accessible alternative component.
